frontend/packages/console-app/src/components/quick-starts/utils/quick-start-context.ts

```diff
@@ -89,8 +91,12 @@ export const useValuesForQuickStartContext = (): QuickStartContextValues => {
           [quickStartId]: getDefaultQuickStartState(totalTasks, QuickStartStatus.IN_PROGRESS),
         };
       });
+      fireTelemetryEvent('Quick Start Initiated', {
+        id: quickStartId,
+        type: 'start',
```

Comment:
Please add the quick start `name` as well to make it easier to understand the event when looking at the event data.

Reply:
@christianvogt Actually, `quickStartId` here is `name`. It seems that `name` is used to identify all the quick starts when the functions are implemented. I can change it to
```
{ name: quickStartId, type: 'start' }
```
Do we need to pass `uid` of the quick starts as `id` here? It's hardly used, so there will be some type changes in the functions because we didn't pass `uid` to this component.

Reply:
The quick start ID is the constant while the name is going to be the human readable value. Quick starts will also be localized so we want to ensure that the ID is a constant value.
The name is useful when looking at the data in segment.

Reply:
After investigation, the adding the name at this point would require some api refactoring and not worth the effort for the initial telemetry release.